package.json

```diff
@@ -133,7 +133,7 @@
     "@kbn/crypto": "link:bazel-bin/packages/kbn-crypto/npm_module",
     "@kbn/mapbox-gl": "link:bazel-bin/packages/kbn-mapbox-gl/npm_module",
     "@kbn/i18n": "link:bazel-bin/packages/kbn-i18n/npm_module",
-    "@kbn/interpreter": "link:packages/kbn-interpreter",
+    "@kbn/interpreter": "link:bazel-bin/packages/kbn-interpreter/npm_module",
```

Comment:
Can we remove `/npm_module` please @jbudz ? It should work correctly after https://github.com/elastic/kibana/pull/100947 be merged